packages/core/addon/mirage/fixtures/reports.js

```diff
@@ -688,5 +688,67 @@ export default [
       ],
       sorts: []
     }
+  },
+  {
+    id: 14,
+    title: 'RequestV2 mmuti-param testing report',
+    createdOn: '2015-04-01 00:00:00',
+    updatedOn: '2015-04-01 00:00:00',
+    authorId: 'navi_user',
+    deliveryRuleIds: [],
+    visualization: {
+      type: 'table',
+      version: 2,
+      metadata: {
+        columnAttributes: {
+          c1: { canAggregateSubtotal: false },
+          c3: { canAggregateSubtotal: false },
+          c2: { canAggregateSubtotal: false }
+        },
+        showTotals: []
+      }
+    },
+    request: {
+      table: 'network',
+      dataSource: 'bardOne',
+      limit: null,
+      requestVersion: '2.0',
+      filters: [
+        {
+          type: 'timeDimension',
+          dataSource: 'bardOne',
+          field: 'network.dateTime',
+          parameters: { grain: 'day' },
+          operator: 'bet',
+          values: ['2015-10-02', '2015-10-14']
```

Comment:
This is what should get saved for v2
```suggestion
          values: ['2015-10-02T00:00:00.000Z', '2015-10-14T00:00:00.000Z']
```